x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_columns.tsx

```diff
@@ -273,9 +332,148 @@ export const useColumns = (
         sortable: true,
         valign: 'middle',
       },
+      ['Baseline rate']: {
+        'data-test-subj': 'aiopsLogRateAnalysisResultsTableColumnBaselineRateChange',
+        field: 'bg_count',
+        name: (
+          <>
+            <FormattedMessage
+              id="xpack.aiops.logRateAnalysis.resultsTable.baselineRateLabel"
+              defaultMessage="Baseline rate"
+            />
+            &nbsp;
+            <EuiIconTip
+              size="s"
+              position="top"
+              color="subdued"
+              type="questionInCircle"
+              className="eui-alignTop"
+              content={baselineRateMessage}
+            />
+          </>
+        ),
+        render: (_, { bg_count: bgCount }) => {
+          if (
+            interval === 0 ||
+            windowParameters === undefined ||
+            buckets === undefined ||
+            isGroupsTable
+          )
+            return NOT_AVAILABLE;
+
+          const { baselineBucketRate } = getBaselineAndDeviationRates(
+            buckets.baselineBuckets,
+            buckets.deviationBuckets,
+            undefined,
+            bgCount
+          );
+
+          return <>{baselineBucketRate}</>;
+        },
+        sortable: false,
+        valign: 'middle',
+      },
+      ['Deviation rate']: {
```

Comment:
Can the `Deviation rate` column be made sortable?

Reply:
Updated in ff861eb3f280d3812b7536fcbabcd06eced039cc